### PR TITLE
added front end config to oidc for redirect http to https and ssl cert policy to LB

### PR DESCRIPTION
**Pull request by smark88**
…policy

### CODE REVIEW

File: charts/oidc-proxy/values.yaml [hunk 1]
```diff
@@ -56,6 +56,7 @@ service:
 
 ingress:
   enabled: true
+  sslPolicy:
```
Comment (nmalfroy) [inline charts/oidc-proxy/values.yaml:59]: Do we not need to give it an empty string or object?

Comment (nmalfroy) [reply charts/oidc-proxy/values.yaml:59]: Ah, I see, with the if statement, probably not

File: charts/oidc-proxy/templates/oidc-frontend-config.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{{- if .Values.ingress.sslPolicy }}
+---
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  name: {{ template "oidc-proxy.fullname" . }}
+  labels:
+    {{- include "oidc-proxy.labels" . | nindent 4 }}
```
Comment (nmalfroy) [inline charts/oidc-proxy/templates/oidc-frontend-config.yaml:8]: Any chance this would need to be different labels than the other oidc component?

**Review by nmalfroy — APPROVED** [review]
Couple questions, but nothing to hold up this PR

File: charts/oidc-proxy/values.yaml (hunk 1 above)
Comment (smark88) [reply charts/oidc-proxy/values.yaml:59]: it's mostly for demonstrating what values can be over written? I've been lazy and haven't written a schema for any of the datarepo charts yet
https://helm.sh/docs/helm/helm_show_values/

**Pull request merged**